src/router/index.js

```diff
@@ -92,13 +92,21 @@ export const constantRoutes = [
     path: '/documentation',
     component: Layout,
     redirect: '/documentation/index',
-    meta: { title: 'documentation', icon: 'documentation', affix: true, breadcrumb: false },
+    meta: {
+      title: 'documentation',
+      icon: 'documentation',
+      breadcrumb: false
+    },
     children: [
       {
         path: 'index',
         component: () => import('@/views/documentation/index'),
         name: 'Documentation',
-        meta: { title: language.t('documentation.releaseNotes'), icon: 'documentation', affix: true, isIndex: true }
+        meta: {
```

Comment:
This is not a really change

Reply:
the affix attribute controls whether it is allowed to be closed while false or not like the dashboard.